packages/components/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@kaizen/components",
+  "version": "1.0.0",
+  "description": "Kaizen component library",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "future",
+    "src",
+    "dist"
+  ],
+  "scripts": {
+    "prepublish": "tsc --project tsconfig.dist.json",
+    "build": "yarn clean && yarn prepublish",
+    "build:watch": "yarn clean && yarn prepublish --watch",
+    "clean": "rimraf 'dist'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cultureamp/kaizen-design-system.git"
+  },
+  "author": "Geoffrey Chong <geoff.chong@cultureamp.com>",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/cultureamp/kaizen-discourse/issues"
+  },
+  "homepage": "https://github.com/cultureamp/kaizen-design-system#readme",
+  "peerDependencies": {
+    "@kaizen/design-tokens": "^2.10.3 || ^3.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0",
```

Comment:
Would the design-tokens be moved into this package at some point? So that they don't need to be a peerDep.

Reply:
I did this cause the yarn error log seemed to complain about this...

Reply:
But yeah it will move to this package, although not sure how the import experience will be for those just yet.